.github/workflows/update-flake-lock.yml

```diff
@@ -0,0 +1,42 @@
+name: Update Nix Flake lockfile
+
+on:
+  # Enable option to manually run the action:
+  workflow_dispatch:
+
+  # Run every Sunday:
+  schedule:
+    - cron: 0 0 * * 0
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: cachix/install-nix-action@v20
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
```

Comment:
Can you add a comment, why unstable?

As an user, I'd rather use stable by default. On the other hand we're talking about a CI job so maybe you're right it's better to use unstable for that, so it's more likely to uncover any brittle code and also it alerts us of upcoming breakage sooner.

Reply:
The channel set up by this GH action is irrelevant since we're strictly using only the nixpkgs version specified by the `flake.lock` file. If amended this commit to add a comment explaining this.